### PR TITLE
Add fastlane bundle 1.0

### PR DESCRIPTION
**Pull request by milch**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/fastlane.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+cask 'fastlane' do
+  VERSION_NUMBER = '1.0'.freeze
+
+  version VERSION_NUMBER
```
Comment (Numbermaniac) [inline Casks/fastlane.rb]: Why not just `version '1.0'`, and then use `#(version}` in the URL? Then `VERSION_NUMBER` isn't needed.

**Comment by miccal** [comment]
This seems to be a CLI-only open-source tool, so it should first be submitted to [Homebrew](https://github.com/Homebrew/homebrew). Please see the [rejected casks document](https://github.com/caskroom/homebrew-cask/blob/master/doc/faq/rejected_casks.md) for more information.

**Comment by milch** [comment]
@miccal Thanks for your review. However, I don't think Homebrew would accept this. Basically everything that they state in their [Acceptable Formulae](https://github.com/Homebrew/brew/blob/master/docs/Acceptable-Formulae.md) docs is not fulfilled by this fastlane bundle, specifically:

- **We try hard to avoid dupes in Homebrew/homebrew**: `fastlane` is a ruby gem
- **We don’t like tools that upgrade themselves**: `fastlane` updates itself
- **We don’t like binary formulae**: The way this bundle is structured means it contains a binary release of ruby, gems, and other tools. The fastlane gem itself that is provided here is also a prebuilt version. 
- **Bindings**: Same here, `fastlane` is a ruby gem.

File: Casks/fastlane.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'fastlane' do
+  version '1.0'
+  sha256 'f642cdae3d4841f85d4a9260866361d136fad82974cf5905629d48f70d86f294'
+
+  # kits-crashlytics-com.s3.amazonaws.com/fastlane/ was verified as official when first introduced to the cask
+  url "https://kits-crashlytics-com.s3.amazonaws.com/fastlane/standalone/bundle-#{version}.zip"
+  name 'fastlane'
+  homepage 'https://fastlane.tools'
+
```
Comment (jawshooah) [inline Casks/fastlane.rb]: Please use `"#{staged_path}/install"` instead of a relative path.

File: Casks/fastlane.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  installer script: './install', args: ['-p', '-u', '-b'], sudo: false
```
Comment (jawshooah) [inline Casks/fastlane.rb]: Please use `"#{staged_path}/install"` instead of a relative path.

**Comment by milch** [comment]
Thanks for the review, @jawshooah ... Just addressed your feedback!

**Comment by KrauseFx** [comment]
🎉 Thanks everyone 🎉

**Pull request merged**